src/lang/pl.rs

```diff
@@ -312,8 +312,8 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("Keep RustDesk background service", "Zachowaj usługę RustDesk w tle"),
         ("Ignore Battery Optimizations", "Ignoruj optymalizację baterii"),
         ("android_open_battery_optimizations_tip", "Jeśli chcesz wyłączyć tę funkcję, przejdź do następnej strony ustawień aplikacji RustDesk, znajdź i wprowadź [Bateria], odznacz [Bez ograniczeń]"),
-        ("Start on Boot", ""),
-        ("Start the screen sharing service on boot, requires special permissions", ""),
+        ("Start on Boot", "Autostart"),
```

Comment:
Is this PL? :)

Reply:
yes

Reply:
I mean I'm native and this is correct and understandable